mlflow/gateway/providers/anthropic.py

```diff
@@ -104,7 +104,10 @@ def __init__(self, config: RouteConfig) -> None:
         if config.model.config is None or not isinstance(config.model.config, AnthropicConfig):
             raise TypeError(f"Invalid config type {config.model.config}")
         self.anthropic_config: AnthropicConfig = config.model.config
-        self.headers = {"x-api-key": self.anthropic_config.anthropic_api_key}
+        self.headers = {
+            "x-api-key": self.anthropic_config.anthropic_api_key,
+            "anthropic-version": "2023-06-01",
```

Comment:
Could we make anthropic-version a config instead of hard-coded here? We could put it into AnthropicConfig

Reply:
+1

Reply:
The fix PR is merged https://github.com/mlflow/mlflow/pull/11056 We could close this PR